AnnoyancesFilter/sections/popups.txt

```diff
@@ -120,6 +120,9 @@ sodapdf.com##.be-tooltip
 !######### Subscription popups add to subscriptions.txt #########
 !
 media.rakuten-sec.net##.m-vote-banner
+shopee.*#$##shopee-banner-popup-stateful { display: none !important; }
```

Comment:
`shopee-banner-popup-stateful` is the tag here. Third `#` makes the rule invalid.